tools/build-sunder-debian-packages.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+set -u
+
+
+SUNDER_CODE=/sunder
+
+if [[ ! -d "${SUNDER_CODE}" ]]; then
+    printf "${SUNDER_CODE} directory not found!\\n"
+    exit 1
+fi
+
+icns2png --extract --output "${SUNDER_CODE}/build/icons/" "${SUNDER_CODE}/build/icon.icns"
+npm install
```

Comment:
I suggest we `npm install` in the Dockerfile. It is a time-consuming step and in general I would expect the Sunder code to change more frequently than its underlying dependencies, so it will probably be worthwhile for Docker to cache the results of `npm install` in a layer.

References:
* http://bitjudo.com/blog/2014/03/13/building-efficient-dockerfiles-node-dot-js/ (explains basic concept)
* https://nodejs.org/en/docs/guides/nodejs-docker-webapp/ (example code includes `package-lock.json` for NPM v5+).

Reply:
Yes! `npm install` downloads a mammoth amount of dependencies, so very happy to cache that.